st2api/tests/unit/controllers/v1/test_timers.py

```diff
@@ -91,13 +97,13 @@ def setUpClass(cls):
         TestTimersController.MODELS = loader.save_fixtures_to_db(
             fixtures_pack=PACK, fixtures_dict=FIXTURES)['triggers']
 
-    def test_get_one_with_id(self):
+    def test_timerscontroller_get_one_with_id(self):
         model = TestTimersController.MODELS['interval1.yaml']
         get_resp = self._do_get_one(model.id)
         self.assertEqual(get_resp.status_int, 200)
         self.assertEqual(get_resp.json['parameters'], model['parameters'])
 
-    def test_get_one_with_ref(self):
+    def test_timerscontroller_get_one_with_ref(self):
```

Comment:
Yeah, I like this naming scheme more - easier and fast to see in Circle CI output which test failing with a quick glace.